packages/client/hmi-client/src/components/data-explorer/search-results-list.vue

```diff
@@ -170,7 +170,9 @@ const resultsText = computed(() => {
 });
 </script>
 
-<style scoped>
+<style scoped lang="scss">
+@import '@/assets/css/theme/variables';
+
```

Comment:
We use the CSS custom properties to avoid doing this.
Please use them instead 😃

Reply:
Sorry, not sure I understand. We want to use standard css variables like `var(--something)`? Then I would have to define that in `/styles/variables.css`? Any way I can leverage the existing variable?

Reply:
Nevermind I see how it is done with existing variables!